### DISCUSSION TITLE
Custom domain from another provider

### DISCUSSION THREAD

**Discussion by JonCognioDigital** [vercel/vercel#5586] [General]
Hi all,

We're thinking of moving our Next.js app to Vercel and I can see info on buying a domain from Vercel or transferring a domain to Vercel but I can't find any info on using an existing domain bought elsewhere. Is it possible to just set the nameservers, A records etc to point at Vercel? If so, are there any instructions anywhere?

**Comment by mcsdevv** [comment]
Hello @jon64digital, can I ask the resources you've been looking at so far?

For reference, the documentation you need can be found [here](https://vercel.com/docs/custom-domains).

**Comment by JonCognioDigital** [comment]
Yes, that's what I've been looking at.

Just before the first heading it says "If you want to migrate an existing domain to Vercel, read our Knowledge Base article on Zero-Downtime Migration for DNS." which sounds like the Domain has to be "migrated" as in moved, to Vercel, although on re-reading it maybe it means migrated as in re-pointed?

The next section is "Adding a domain" which starts with "If you don't own a domain yet, you can purchase it with Vercel. Then, once you own a domain, adding it to a project is very straightforward:" so I scanned down further and found sections on "redirecting" and "moving" domains, neither of which is what I want to do.

So, having re-read it I can see that the first section is about adding a domain even if you haven't bought it through Vercel, but that wasn't obvious to me. Hope that helps.
